src/python/grpcio/grpc/__init__.py

```diff
@@ -1219,12 +1219,14 @@ def server(thread_pool, handlers=None):
       only handlers the server will use to service RPCs; other handlers may
       later be added by calling add_generic_rpc_handlers any time before the
       returned Server is started.
+    args: A grpc._cython.ChannelArgs to be used to provide configuration options
```

Comment:
grpc._cython.ChannelArgs is an internal class.  Could you change this type (and name) to match the options parameter on client side channel creation (i.e ```secure_channel()```), and do the needed conversion in src/python/grpcio/grpc/_server.py ?